test/wrong_command.c

```diff
@@ -5,5 +5,9 @@ int main(void) {
 		"This is not the command you are looking for.\n"
 		"You probably want to run 'python3 -m pytest test/' instead"
 		"\e[0m\n");
+#if defined(MESON_IS_SUBPROJECT)
```

Comment:
Why not `#ifdef`?

Reply:
It's a force of habit :) I can fix that too. let me close this pull request make al the changes and see if I can do it more appropriately :)